great_expectations/checkpoint/checkpoint.py

```diff
@@ -55,19 +55,28 @@
 from great_expectations.util import (
     deep_filter_properties_iterable,
     filter_properties_dict,
+    load_class,
 )
 from great_expectations.validation_operators import ActionListValidationOperator
 from great_expectations.validation_operators.types.validation_operator_result import (
     ValidationOperatorResult,  # noqa: TCH001
 )
-from great_expectations.validator.validator import Validator
 
 if TYPE_CHECKING:
     from great_expectations.data_context import AbstractDataContext
+    from great_expectations.validator.validator import Validator
+
 
 logger = logging.getLogger(__name__)
 
 
+def _get_validator_class() -> Type[Validator]:
+    """Using this function helps work around circular import dependncies."""
+    module_name = "great_expectations.validator.validator"
+    class_name = "Validator"
```

Comment:
We should be using dunders here I think - `Validator.__class__.__name__` and the same for module

Reply:
Why not directly just instantiate a `Validator`?

Reply:
@cdkini This is the inline import -- the `load_class()` method is the actual implementation for the inline import, which is syntactic convenience.

Reply:
The idea here is to avoid circular dependency, so `Validator.__class__.__name__` would not work (we are trying to avoid depending on the actual `Validator`, except for `TYPE_CHECKING`.  Thanks.